AppCenterAuth/AppCenterAuth/Internals/Models/MSB2CAuthority.m

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSB2CAuthority.h"
+#import "../Util/MSAuthConstants.h"
+
+@implementation MSB2CAuthority
+
+static NSString *const kMSType = @"type";
```

Comment:
These should be reused from `MSAADAuthority` - put them in a common place

Reply:
Can you please suggest this common place? Should I add this to MSAuthConstants?